Doc/library/argparse.rst

```diff
@@ -1695,9 +1672,22 @@ Sub-commands
    :class:`ArgumentParser` supports the creation of such subcommands with the
    :meth:`!add_subparsers` method.  The :meth:`!add_subparsers` method is normally
    called with no arguments and returns a special action object.  This object
-   has a single method, :meth:`~_SubParsersAction.add_parser`, which takes a
-   command name and any :class:`!ArgumentParser` constructor arguments, and
-   returns an :class:`!ArgumentParser` object that can be modified as usual.
+   has a single method, :meth:`~_SubParsersAction.add_parser`:
+
+   .. method:: _SubParsersAction.add_parser(name, *, help=None, aliases=None, deprecated=False, **kwargs)
+
+      Creates and returns a new :class:`!ArgumentParser` object for the
+      subcommand *name*.
+
+      The *name* argument is the name of the sub-command.
+      The *help* argument provides a short description for this sub-command. If provided, it will be listed next to the command in the main parser's help message (e.g., ``PROG --help``).
```

Comment:
```suggestion
      The *help* argument provides a short description for this subcommand. If provided, it will be listed next to the command in the main parser's help message (e.g., ``PROG --help``).
```